ls1intum/Artemis
# [Bugfix] Assessment Dashboard Tutor Leaderboard ISE

### Checklist
### Motivation and Context


Fixes #2805 

### Description

- Renamed 'max_score' to 'max_points'

### Steps for Testing


1. Log in to Artemis
2. Navigate to Course Administration
3. Go to Assessment Dashboard and see if the error is still thrown like described in attached issue
4. Check that the amount of assessed/unassessed submission and open/evaluated complaints or more feedback requests are also correct.

### Test Coverage



Nothing changed

### Screenshots


<img width="1537" alt="image" src="https://user-images.githubusercontent.com/41108487/106642985-5a9b8a80-6589-11eb-97f7-914edca97e02.png">
